### ISSUE TITLE
localForage is not assignable to type PersistentStorage

### ISSUE THREAD

**Issue by MatthiasKunnen** [apollographql/apollo-cache-persist#399]
PR #388 broke assignment of localForage.

Error:
```
- error TS2322: Type 'LocalForage' is not assignable to type 'PersistentStorage'.
  The types returned by 'setItem(...)' are incompatible between these types.
    Type 'Promise<string>' is not assignable to type 'void | Promise<void>'.
      Type 'Promise<string>' is not assignable to type 'Promise<void>'.
        Type 'string' is not assignable to type 'void'.

32         storage: localForage,
           ~~~~~~~

  node_modules/apollo3-cache-persist/lib/types/index.d.ts:14:5
    14     storage: PersistentStorage;
           ~~~~~~~
    The expected type comes from property 'storage' which is declared here on type 'ApolloPersistOptions<NormalizedCacheObject>'
```

**Comment by MatthiasKunnen** [comment]
This is still an issue in 0.9.1:
```
error TS2322: Type 'LocalForage' is not assignable to type 'PersistentStorage'.
  The types returned by 'setItem(...)' are incompatible between these types.
    Type 'Promise<string>' is not assignable to type 'void | Promise<void>'.
      Type 'Promise<string>' is not assignable to type 'Promise<void>'.
        Type 'string' is not assignable to type 'void'.

107             storage: localForage,
                ~~~~~~~

  node_modules/apollo3-cache-persist/lib/types/index.d.ts:14:5
    14     storage: PersistentStorage;
           ~~~~~~~
    The expected type comes from property 'storage' which is declared here on type 'ApolloPersistOptions<NormalizedCacheObject>'

```

**Comment by wodCZ** [comment]
Closing this, since dedicated LocalForage wrapper was created and both mentioned TypeScript errors aren't relevant anymore.
Please create a new issue if you're having troubles with LocalForage wrapper in the latest release.

**Comment by MatthiasKunnen** [comment]
@wodCZ, could you link to the respective PR/commit and a code sample if one exists?

**Comment by wodCZ** [comment]
@MatthiasKunnen yes, that's the PR, seems to be released since 0.9.0.

Note that the new API requires you to wrap the localForage in `LocalForageWrapper`

```ts

import { persistCache, LocalForageWrapper } from 'apollo3-cache-persist';
// ...
await persistCache({
  cache,
  storage: new LocalForageWrapper(localForage),
});
```

Not directly like in your last comment (`storage: localForage`)
